dev.yml

```diff
@@ -1,16 +1,15 @@
 ---
-- name: "WordPress Server: Install LEMP Stack with PHP 7.0 and MariaDB MySQL"
+- name: "WordPress Server: Install LEMP Stack with PHP 7.0 and chosen MySQL"
```

Comment:
While you're here, let's just simplify this to: "WordPress Server: Install LEMP stack"